--- dbh-5.0.7/configure.ac.orig	2013-07-18 22:55:37.000000000 +0200
+++ dbh-5.0.7/configure.ac	2014-01-14 22:00:51.501153224 +0100
@@ -161,7 +161,7 @@
 
 ac_uname_s=`uname -s`
 echo "system: $ac_uname_s"
-AM_CONDITIONAL(WANT_FREEBSD, [test "$ac_uname_s"=FreeBSD || "$ac_uname_s"=OpenBSD])
+AM_CONDITIONAL(WANT_FREEBSD, [test "$ac_uname_s" = FreeBSD -o "$ac_uname_s" = OpenBSD])
 
 AC_CHECK_LIB([pthread],[pthread_create])
 AC_CHECK_LIB([pthread],[pthread_cond_signal],[ac_pthread_ok=yes],[ac_pthread_ok=no])
